tpm2/snapmodel_policy_test.go

```diff
@@ -418,7 +418,9 @@ func (s *snapModelMeasureSuite) testMeasureSnapModelToTPMTest(c *C, data *testMe
 	c.Assert(err, IsNil)
 
 	for _, s := range pcrSelection {
-		snapModelDigest, err := ComputeSnapModelDigest(s.Hash, data.model)
+		snapModelDigest, err := ComputeSnapModelDigest(func() (SnapModelHasher, error) {
+			return &GoSnapModelHasher{s.Hash.NewHash()}, nil
```

Comment:
this also shows that old code and new code compute the same things?

Reply:
Indirectly, yes. The tests for AddSnapModelProfile have hardcoded digests in the test so if this code computed a different digest, those tests would fail.